src/Evolution/Executables/GrMhd/ValenciaDivClean/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 # Distributed under the MIT License.
 # See LICENSE.txt for details.
-
```

Comment:
Please don't add or delete empty lines in various places :D what I usually do is look over the diff in VSCode and drop changes that are unrelated to the PR.